components/Card/src/Card/CaseCard.tsx

```diff
@@ -2,105 +2,44 @@ import React, { AnchorHTMLAttributes, ComponentType } from 'react';
 import { ArrowRightIcon } from '@gemeente-denhaag/icons';
 import { BasicLink } from '@gemeente-denhaag/link';
 import { Paragraph } from '@gemeente-denhaag/typography';
-import clsx from 'clsx';
 import '../index.scss';
-import Card from './Card';
-import CardBackground from './CardBackground';
-import CardActions from './CardActions';
-import CardContent from './CardContent';
-import CardTextWrapper from './CardTextWrapper';
-import CardWrapper from './CardWrapper';
-import CardDateWrapper from './CardDateWrapper';
-import CardDate from './CardDate';
-import { CardAction } from './CardAction';
-import { formatDate, FormatDateLabels, longDateOptions } from '@gemeente-denhaag/utils';
+import Card from './CaseCardBase';
+import Footer from './CaseCardFooter';
+import Action from './CaseCardAction';
+import Wrapper from './CaseCardWrapper';
+import Background from './CaseCardBackground';
+import Context from './CaseCardContext';
+import Container from './CaseCardContainer';
 
 export interface CaseCardProps {
-  /**
-   * Determines the title of the card
-   */
   title: string;
-
-  /**
-   * Determines the subtitle of the card
-   */
   subTitle?: string;
-
-  /**
-   * Determines the date shown on the card
-   */
-  date?: string;
-
-  /**
-   * Determines the date format shown on the card
-   */
-  dateLabels?: FormatDateLabels;
-
-  /**
-   * Determines the date format shown on the card
-   */
-  locale?: string;
-
-  /**
-   * Determines the url the card points to
-   */
+  context?: React.ReactNode;
   href?: string;
-
-  /**
-   * Determines the card color
-   */
-  active?: boolean;
-
-  /**
-   * Custom Link component used for single-page apps.
-   */
+  appearance?: 'archived' | 'list';
```

Comment:
Ik zou zelf denk ik een 'default' toevoegen als optie, en die als default toevoegen, maar dat is persoonlijke voorkeur, dus kun je negeren. ;)